data_steward/tools/create_tier.py

```diff
@@ -0,0 +1,130 @@
+# Python imports
+import argparse
+import logging
+import re
+
+# Third party imports
+
+# Project imports
+
+LOGGER = logging.getLogger(__name__)
+
+
+def validate_tier_param(tier):
+    """
+    helper function to validate the tier parameter passed is either 'controlled' or 'registered'
+
+    :param tier: tier parameter passed through from either a list or command line argument
+    :return: nothing, breaks if not valid
+    """
+    if tier.lower() not in ['controlled', 'registered']:
+        LOGGER.error(
+            f"Parameter ERROR: {tier} is an incorrect input for the tier parameter, accepted: controlled or "
+            f"registered")
+
+
+def validate_deid_stage_param(deid_stage):
+    """
+    helper function to validate the deid_stage parameter passed is correct, must be 'deid', 'base' or 'clean'
+
+    :param deid_stage: deid_stage parameter passed through from either a list or command line argument
+    :return: nothing, breaks if not valid
+    """
+    if deid_stage not in ['deid', 'base', 'clean']:
```

Comment:
Consider moving this list a a module level list so you can reuse it in the argument parser setup.

```suggestion
    if deid_stage not in DEID_STAGE_LIST:
```

where `DEID_STAGE_LIST = ['deid', 'base', 'clean']`.